.vscode/launch.json

```diff
@@ -0,0 +1,24 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Pytest",
+            "type": "debugpy",
+            "request": "launch",
+            "module": "pytest",
+            "justMyCode": false,
+            "cwd": "${fileDirname}"
+        },
+        {
+            "name": "Python Debugger: Current File with Arguments",
+            "type": "debugpy",
+            "request": "launch",
+            "program": "${file}",
+            "console": "integratedTerminal",
+            "args": "${command:pickArgs}"
+        }
+    ]
+}
```

Comment:
Would you, please, drop this file from the pull request completely (i.e., please, do an interactive rebase and remove it from the history)? Thanks!

I am not 100% convinced these files should be part of the repository at all (and honestly, M$ again decided to break standards again by calling it JSON and adding a non-standard comments into it).